keyboards/wuque/ikki68_aurora/readme.md

```diff
@@ -0,0 +1,20 @@
+# ikki68_aurora
+
+![ikki68_aurora](https://cdn.shopify.com/s/files/1/0403/9809/4489/products/GMK_Mizu_FK5225_2021-Jan-28_01-50-51AM-000_CustomizedView25613364763.jpg?v=1611994134)
+​
+Hope this kit is the same as ikki, every rebirth will be stronger
+More Info at[wuquestudio](https://shop.wuquestudio.com/).
+​
+* Keyboard Maintainer: [wuquestudio](https://shop.wuquestudio.com)
+* Hardware Supported: ikki68 aurora Standard
+* Hardware Availability: [wuquestudio](http://aurora.wuquestudio.com/)
+​
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to get into the bootloader?

Reply:
Sorry, can you give me an example? Because I haven't saw it it the <readme.md> template.

Reply:
The problem has been solved.